src/twisted/runner/test/test_procmon.py

```diff
@@ -547,3 +598,56 @@ def test_stopServiceCleanupScheduledRestarts(self):
         # all pending process restarts.
         self.assertEqual(self.pm.protocols, {})
 
+
+
+class DeprecationTests(unittest.SynchronousTestCase):
+
+    """
+    Tests that check functionality that should be deprecated is deprecated.
+    """
+
+    def setUp(self):
+        """
+        Create reactor and process monitor
+        """
+        self.reactor = DummyProcessReactor()
+        self.pm = ProcessMonitor(reactor=self.reactor)
+
+
+    def test_toTuple(self):
+        """
+        _Process.toTuple is deprecated
```

Comment:
I think that this needs extra info. I think that I get what you are doing here, but is better to be explict.
The test name and docstring talks about **toTuple** but I don't see any call to toTuple here.

I don't that we should write the docsting assuming that people will know that _Process is a magic class produced by `attr` :)